apps/console/src/features/connections/components/wizards/authenticator-create-wizard.tsx

```diff
@@ -212,22 +218,33 @@ export const AuthenticatorCreateWizard: FunctionComponent<AddAuthenticatorWizard
      * @param identityProvider - Identity provider data.
      */
     const handleWizardFormFinish = (identityProvider: ConnectionInterface): void => {
-        const authenticator = identityProvider?.federatedAuthenticators?.authenticators.find((a) =>
-            a.authenticatorId === identityProvider?.federatedAuthenticators?.defaultAuthenticatorId);
+        const authenticator: FederatedAuthenticatorListItemInterface =
```

Comment:
Here, we are filtering the authenticator list to find the default authenticator, aren't we?

If yes, shall we update the variable name to something like `defaultAuthenticator`?

Reply:
No here it is not the default authenticator always. The method handling new authenticator is reason for doing this.